libweb3jsonrpc/Test.cpp

```diff
@@ -32,6 +33,29 @@ using namespace jsonrpc;
 
 Test::Test(eth::Client& _eth): m_eth(_eth) {}
 
+bool Test::test_setChainParams(const Json::Value &param1)
```

Comment:
You mean should it belong to test section?
I think so. Normally node is started with either --genesis or --config section

This feature is experimental and for test purpose only.